usage-advanced.py

```diff
@@ -2,226 +2,215 @@
 
 import dash_player
 import dash
-import dash_html_components as html
-import dash_core_components as dcc
-from dash.dependencies import Input, Output, State
+from dash import dcc, html, Input, Output, State
```

Comment:
Generally discouraged to import the same package two ways - include `Dash` in the named imports (and `callback` per my other comment) and the `import dash` can be removed.

Reply:
@alexcjohnson I never knew that, thanks for the heads up!